### PR TITLE
[MU4] Sync CI scripts from 3.x to master

### PR DESCRIPTION
**Pull request by igorkorsukov**
Sync CI scripts from 3.x to master

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -114,7 +114,7 @@ debug:
    	  -DBUILD_PORTAUDIO="${BUILD_PORTAUDIO}"              \
    	  -DBUILD_WEBENGINE="${BUILD_WEBENGINE}"              \
    	  -DUSE_SYSTEM_FREETYPE="${USE_SYSTEM_FREETYPE}"      \
-      -DCOVERAGE="${COVERAGE}"                 			\
+          -DCOVERAGE="${COVERAGE}"                 \
    	  -DDOWNLOAD_SOUNDFONT="${DOWNLOAD_SOUNDFONT}"      \
```
Comment (Jojo-Schmitz) [inline Makefile:118]: Looks like a few spaces too many?

File: build/ci/linux/build.sh [hunk 2]
```diff
@@ -68,17 +67,17 @@ echo "=== BUILD ==="
...
-    BUILD_NUMBER=$BUILD_NUMBER \
-    TELEMETRY_TRACK_ID=$TELEMETRY_TRACK_ID \
-    BUILD_UI_MU4=$BUILD_UI_MU4 \
-    BUILD_VST=$BUILD_VST \
-    VST3_SDK_PATH=$VST3_SDK_PATH \
-    BUILD_UNIT_TESTS=ON \
-    MODULE_BUILD_PCH=OFF \
-    $BUILDTYPE
+make CPUS=2 $OPTIONS \
+     MUSESCORE_BUILD_CONFIG=$MUSESCORE_BUILD_CONFIG \
```
Comment (Jojo-Schmitz) [inline build/ci/linux/build.sh:71]: 5 spaces?

File: Makefile.osx [hunk 3]
```diff
@@ -37,30 +37,28 @@ ifneq ($(strip $(shell which xcpretty)),)
...
+	@echo "Make release"; \
+	mkdir build.release; \
+	cd build.release; \
+	cmake -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
+	-DCMAKE_BUILD_TYPE=RELEASE \
+	-DCMAKE_BUILD_NUMBER="${BUILD_NUMBER}" \
+	-DMUSESCORE_BUILD_CONFIG="${MUSESCORE_BUILD_CONFIG}" \
+	-DMUSESCORE_REVISION="${MUSESCORE_REVISION}" \
+	-DTELEMETRY_TRACK_ID="${TELEMETRY_TRACK_ID}" \
+	.. -G Xcode; \
```
Comment (Jojo-Schmitz) [inline Makefile.osx:49]: I'd rather indent all those options to `cmake`

File: Makefile.osx [hunk 4]
```diff
@@ -90,14 +88,19 @@ installdebug:
...
+	@echo "Make ci"; \
+	mkdir build.release; \
+	cd build.release; \
+	cmake -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
+	-DCMAKE_BUILD_TYPE=RELEASE \
+	-DCMAKE_BUILD_NUMBER="${BUILD_NUMBER}" \
+	-DMUSESCORE_BUILD_CONFIG="${MUSESCORE_BUILD_CONFIG}" \
+	-DMUSESCORE_REVISION="${MUSESCORE_REVISION}" \
+	-DTELEMETRY_TRACK_ID="${TELEMETRY_TRACK_ID}" \
+	.. -G Xcode; \
```
Comment (Jojo-Schmitz) [inline Makefile.osx:101]: I'd rather indent all those options to `cmake`
It is done that way further up too

**Comment by Jojo-Schmitz** [comment]
in 3.x you added back mscscore/revisions.h to .gitignore, maybe here too?

File: Makefile (hunk 1 above)
Comment (igorkorsukov) [reply Makefile:118]: fixed

File: build/ci/linux/build.sh (hunk 2 above)
Comment (igorkorsukov) [reply build/ci/linux/build.sh]: fixed

File: Makefile.osx (hunk 3 above)
Comment (igorkorsukov) [reply Makefile.osx]: fixed

File: Makefile.osx (hunk 4 above)
Comment (igorkorsukov) [reply Makefile.osx]: fixed

**Comment by igorkorsukov** [comment]
> in 3.x you added back mscscore/revisions.h to .gitignore, maybe here too?

added

**Pull request merged**